index.html

```diff
@@ -39,77 +39,316 @@
+                I can help you build a product,
+                feature or website Look through
+                some of my work and experience! If 
+                you like what you see and have a 
+                project to be coded,
+                don't hesitate to Contact me.
               </p>
               <br />
-              <div class="h2"></div>
-              <div class="social-links">
-                <ul class="ulist">
-                  <li class="ulist-icon">
-                    <a href="https://twitter.com/dag_mawi111 "
-                      ><img src="./images/twitter.svg" alt="twitter icon"
-                    /></a>
-                  </li>
-                  <li class="ulist-icon">
-                    <a href="https://linkedin.com/in/dagmawi-yohannes-b36488236"
-                      ><img src="./images/Linkedinicon.svg" alt="linkedin icon"
-                    /></a>
-                  </li>
-                  <li class="ulist-icon">
-                    <a href="https://medium.com/dagmawi.yoha03"
-                      ><img src="./images/medium.svg" alt="medium icon"
-                    /></a>
-                  </li>
-                  <li class="ulist-icon">
-                    <a href="https://github.com/dagmawiii03"
-                      ><img src="./images/github.svg" alt="github icon"
-                    /></a>
-                  </li>
-                </ul>
-              </div>
+              <br />
+              <br />
+              <button>Get My Resume</button>
+              <br />
+              <br />
             </section>
-            <section class="pointer">
+          <section class="abt-card">
+            <section class="card">
+              <br />
+              <img src="./images/languages-icon.png" alt="" />
+              <br />
+              <h3>Languages</h3>
+              <br />
               <ul>
-                <li>
-                  <a href=""><i class="fas fa-chevron-down white-color"></i></a>
-                </li>
+                <li>JavaScript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>CSS</li>
               </ul>
             </section>
-          </div>
+          <section class="card">
+            <br />
+            <img src="./images/frameworks-icon.png" alt="" />
+            <br />
+            <h3>Frameworks</h3>
+            <br />
+            <ul>
+              <li>Bootstrap</li>
+              <li>Ruby on rails</li>
+              <li>RSpec</li>
+            </ul>
+          </section>
+          <section class="card">
+            <br />
+            <img src="./images/skills-icon.png" alt="" />
+            <br />
+            <h3>Skills</h3>
+            <br />
+            <ul>
+              <li>Codekit</li>
+              <li>GitHub</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
+          </section>
```

Comment:
**OPTIONAL** - To help improve the semantics of your site, I suggest that you use a `ul` tag to wrap technology categories ("Languages", "Frameworks" and "Skills"), with each category in a corresponding `li` tag. Also the technology in each category should be wrapped in a `ul` tag which each technology wrapped in a corresponding `li` tag. Please fix this 🙏 
```
<ul>
  <li>Languages
    <ul>
      <li>HTML</li>
      <li>CSS</li>
      <li>Javascript</li>
    </ul>
  </li>
  <li>Frameworks</li>
  <li>Skills</li>
</ul>
```